runtime/src/configs/mod.rs

```diff
@@ -231,7 +256,7 @@ impl pallet_message_queue::Config for Runtime {
     // The XCMP queue pallet is only ever able to handle the `Sibling(ParaId)` origin:
     type QueueChangeHandler = NarrowOriginToSibling<XcmpQueue>;
     type QueuePausedQuery = NarrowOriginToSibling<XcmpQueue>;
-    type HeapSize = sp_core::ConstU32<{ 64 * 1024 }>;
+    type HeapSize = sp_core::ConstU32<{ 103 * 1024 }>;
```

Comment:
103 seems like an odd number 🤔

Reply:
It's related to [this PR](https://github.com/paritytech/polkadot-sdk/pull/3952)